web/views/_poll_link.erb

```diff
@@ -1,4 +1,4 @@
-<% if current_path != '' %>
+<% if ['', 'metrics'].exclude?(current_path.split("/").first) %>
```

Comment:
Maybe it would make more sense to configure it from the actons?
E.g. adding @polling_disabled to the /sidekiq and the metric-pages.

Reply:
Yeah, this looks ugly but I don't have a good solution. Maybe a helper like `def pollable?` in web/helpers.rb?

Reply:
Yeah, I will give it a go tomorrow 🙂

Reply:
A helper method looks much cleaner!